core/src/main/scala/kafka/tools/MirrorMaker.scala

```diff
@@ -357,6 +357,8 @@ object MirrorMaker extends Logging with KafkaMetricsGroup {
               trace("Caught NoRecordsException, continue iteration.")
             case _: WakeupException =>
               trace("Caught WakeupException, continue iteration.")
+            case e: KafkaException if (shuttingDown || exitingOnSendFailure) =>
+              trace(s"Ignoring caught KafkaException (${e.getMessage}) during shutdown. sendFailure: $exitingOnSendFailure.")
```

Comment:
Since it's trace level anyway, maybe we should just print the stacktrace instead of just the message.